### DISCUSSION TITLE
Welcome to prisma Discussions!

### DISCUSSION THREAD

**Discussion by SmartCoding88** [khouloudAmamouHaddad/prisma#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by khouloudhaddad** [comment]
[https://www.prisma.io/docs/concepts/components/prisma-client/crud]

**Comment by khouloudhaddad** [comment]
https://www.prisma.io/docs/concepts/database-connectors/mysql

**Reply by SmartCoding88** [reply]
https://www.prisma.io/docs/concepts/database-connectors/mongodb

**Reply by khouloudhaddad** [reply]
https://www.prisma.io/client
